### ISSUE TITLE
[CI] MixedClusterClientYamlTestSuiteIT#{p0=indices.get/10_basic/Should return an exception when querying invalid indices} test failure

### ISSUE THREAD

**Issue by martijnvg** [elastic/elasticsearch#26686]
Reproduce line: `gradle :qa:mixed-cluster:v5.6.1-SNAPSHOT#mixedClusterTestRunner -Dtests.seed=3CEAE6E119461AC2 -Dtests.class=org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT -Dtests.method="test {p0=indices.get/10_basic/Should return an exception when querying invalid indices}" -Dtests.security.manager=true -Dtests.locale=sr -Dtests.timezone=America/Rainy_River` (doesn't reproduce)

```
 | MixedClusterClientYamlTestSuiteIT.test {p0=indices.get/10_basic/Should return an exception when querying invalid indices} <<< FAILURES!
   > Throwable #1: java.lang.AssertionError: Failure at [indices.get/10_basic:166]: expected [400] status code but api [indices.get] returned [404 Not Found] [{"error":{"root_cause":[{"type":"index_not_found_exception","reason":"no such index","index_uuid":"_na_","resource.type":"index_or_alias","resource.id":"_foo","index":"_foo","stack_trace":"[_foo] IndexNotFoundException[no such index]\n\tat 
   > Expected: <400>
  2> NOTE: leaving temporary files on disk at: /var/lib/jenkins/workspace/elastic+elasticsearch+6.x+oracle-periodic/qa/mixed-cluster/build/testrun/v5.6.1-SNAPSHOT#mixedClusterTestRunner/J0/temp/org.elasticsearch.backwards.MixedClusterClientYamlTestSuiteIT_3CEAE6E119461AC2-001
  2> NOTE: test params are: codec=Asserting(Lucene70): {}, docValues:{}, maxPointsInLeafNode=607, maxMBSortInHeap=5.852085763359803, sim=RandomSimilarity(queryNorm=true): {}, locale=sr, timezone=America/Rainy_River
  2> NOTE: Linux 4.4.62-18.6-default amd64/Oracle Corporation 1.8.0_144 (64-bit)/cpus=4,threads=1,free=361172760,total=519045120
  2> NOTE: All tests run in this JVM: [MixedClusterClientYamlTestSuiteIT]
   >      but: was <404>
   > 	at __randomizedtesting.SeedInfo.seed([3CEAE6E119461AC2:B4BED93BB7BA773A]:0)
   > 	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:349)
   > 	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.test(ESClientYamlSuiteTestCase.java:329)
   > 	at java.lang.Thread.run(Thread.java:748)
   > Caused by: java.lang.AssertionError: expected [400] status code but api [indices.get] returned [404 Not Found] [{"error":{"root_cause":[{"type":"index_not_found_exception","reason":"no such index","index_uuid":"_na_","resource.type":"index_or_alias","resource.id":"_foo","index":"_foo","stack_trace":"[_foo] IndexNotFoundException[no such index]\n\tat org.elasticsearch.cluster.metadata.IndexNameExpressionResolver$WildcardExpressionResolver.infe(IndexNameExpressionResolver.java:676)\n\tat
\n\tat java.lang.Thread.run(Thread.java:748)\n"},"status":404}]
   > Expected: <400>
   >      but: was <404>
   > 	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
   > 	at org.elasticsearch.test.rest.yaml.section.DoSection.assertStatusCode(DoSection.java:307)
   > 	at org.elasticsearch.test.rest.yaml.section.DoSection.execute(DoSection.java:240)
   > 	at org.elasticsearch.test.rest.yaml.ESClientYamlSuiteTestCase.executeSection(ESClientYamlSuiteTestCase.java:345)
   > 	... 37 more
```

Build urls: 
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+oracle-periodic/633/consoleText
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.0+intake/270/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+multijob-windows-compatibility/155/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+multijob-unix-compatibility/os=oraclelinux/155/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+multijob-unix-compatibility/os=fedora/155/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+multijob-unix-compatibility/os=amazon/155/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+oracle-periodic/629/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+periodic/24/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+java9-periodic/636/console
* https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+6.x+java9-periodic/639/console

**Comment by jasontedor** [comment]
Closed by 5dd476feb5cd9f96f7f517dd50ce5bac6df101cb

**Comment by hub-cap** [comment]
@jasontedor @martijnvg thanks, sorry about this.
